docs/docsite/rst/playbook_guide/playbook_pathing.rst

```diff
@@ -37,7 +39,26 @@ Specifically, Ansible tries to find the file
 3. Like 1, in the current task file's directory.
 4. Like 1, in the current play file's directory.
 
-Ansible does not search the current working directory. (The directory you're in when you execute Ansible.) Also, Ansible will only search within a role if you actually included it with an `include_role` or `import_role` task or a dependency. If you instead use `include`, `include_task` or `import_task` to include just the tasks from a specific file but not the full role, Ansible will not search that role in steps 1 and 2.
+Ansible does not search the current working directory. (The directory you're in when you execute Ansible.).
+
+Resolving local relative tasks paths
+------------------------------------
+
+Ansible will only search for task files within a role if you actually included them with an `include_role` or `import_role`. If you instead use `include`, `include_task` or `import_task` to include just the tasks from a specific file but not the full role, Ansible will only search for the task in the current role.
```

Comment:
```suggestion
Ansible searches for task files within a role only if you include them with an `include_role` or `import_role` statement. If you use `include`, `include_task`, or `import_task` statement to include only the tasks from a specific file, but not the full role, Ansible searches for the task in the current role.
```

Reply:
Some nits for grammar and clarity. In general it's better to use present simple tense instead of future.

Reply:
so, this has always been misleading as if the include/import_tasks is within a role itself, that role's context will be used. For example the RoleB's context will not be used if from a play or roleA you include/import_tasks from roleB. While the roleB context will not be used, in the 2nd case, roleA's context is still valid.